### PR TITLE
Add JMX file for JMeter testing

### PR DESCRIPTION
**Pull request by AngeloDanducci**
# Changes

- Add JMeter test plan for performance testing and update README

/kind documentation

Necessary but not sufficient for [this issue](https://github.com/knative-sandbox/async-component/issues/62) so I will avoid auto close with the `Fixes` syntax.

### CODE REVIEW

**Comment by maximilien** [comment]
Please resolve not README conflict

File: test/JMeter/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+**Stress Test Guide:**
+
+Stress testing on the ingress was performed using [JMeter](https://jmeter.apache.org/download_jmeter.cgi).
+Initial testing was performed in GUI mode, however if pushing the limits it can be run from CLI as well.
+
+In order to stress test open the `PerformanceTest.jmx` file with JMeter and edit these properties:
```
Comment (maximilien) [inline test/JMeter/README.md:6]: Add a link to the file

Comment (maximilien) [reply test/JMeter/README.md:6]: And to JMeter and please specify the version used

File: test/JMeter/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+- In HTTP Request Defaults, edit `Server Name or IP` field to point to your helloworld application.
```
Comment (maximilien) [inline test/JMeter/README.md:7]: I suggest including UI screen shots with annotation. And also including CLI commands as alternative. Ideally CLI but if GUI is more intuitive then include both please.

File: test/JMeter/README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    Recall that you can find this by using `kubectl get kservice helloworld-sleep
```
Comment (maximilien) [inline test/JMeter/README.md:8]: I think it's more involved then this one `kubectl` command. You likely need to output in JSON and parse that to extract the IP. Search for "Knative Service IP address" and you will find various help on the topic... including kubectl examples.

File: test/JMeter/README.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  `
+  
+- You may also wish to edit the Graph Results and Summary Report output paths.
```
Comment (maximilien) [inline test/JMeter/README.md:11]: Show an example of the graph here

File: test/JMeter/README.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+After editing these fields simply run the test plan.  
+
+The test plan will spin up 100 threads with 10 loops of GET requests to the helloworld application and log the outcome.
```
Comment (maximilien) [inline test/JMeter/README.md:15]: Perhaps mention how to change these parameters. I assume changing the JMX file?

**Review by maximilien — CHANGES_REQUESTED** [review]
Various feedback. Please address and tag me when ready for another review. Thanks.

**Comment by maximilien** [comment]
/retest
@AngeloDanducci you need to resolve conflict and push changes again

**Comment by AngeloDanducci** [comment]
Merge conflict solved, updated information should be good for a re-review.

**Review by maximilien — APPROVED** [review]
/approve
/lgtm

**Pull request merged**